Lesson1/Lesson_1.3/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+
+namespace Lesson_1._3
+{
+	class Program
+	{
+		static void Main(string[] args)
+		{
+			Console.WriteLine("Input first line");
+			string firstLine = Console.ReadLine().ToUpper();
+
+			//Console.WriteLine($"{firstLine}");
```

Comment:
Commentaries in the code should notice something for developer or should have TODO prefix if you wanna return to it later, any debug code should be removed